htmlelements-matchers/src/main/java/ru/yandex/qatools/htmlelements/matchers/decorators/ConditionMatcherDecorator.java

```diff
@@ -40,9 +49,10 @@ protected void describeMismatchSafely(T item, Description mismatchDescription) {
     }
 
     @Factory
-    public static <T, E> Matcher<T> decorateMatcherWithCondition(final Matcher<? super T> matcher,
-                                                                 final E itemToMatchCondition,
-                                                                 final Matcher<? super E> condition) {
+    public static <T, E> Matcher<T> decorateMatcherWithCondition(
```

Comment:
thanks for the video, but we prefer previous variant while it can be placed in 120 symbols